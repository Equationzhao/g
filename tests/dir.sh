output="$(g --no-update -term-width 200 --no-config --icons --permission --size -d tests/test_data )"
echo "$output" | diff - tests/dir.stdout
