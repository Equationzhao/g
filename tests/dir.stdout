[0;34m[1md[0;33m[1mr[0;31m[1mw[0;32m[1mx[0;33m[1mr[0;90m-[0;32m[1mx[0;33m[1mr[0;90m-[0;32m[1mx[0m [38;2;20;207;100m1.0 KiB[0m [0;94mî—¿ test_data[0m  
